### ISSUE TITLE
Element Call Scrennshare broken since Nightly 2023062301

### ISSUE THREAD

**Issue by YourSandwich** [vector-im/element-web#25714]
### Steps to reproduce

1. Enable Element Call feature in Labs
2. Join an Element Call enable Room
3. Press on the screenshare button

#### What did you expect?
Nothing happens

#### What happened instead?
A pop-up asks what screen or application to use

### Operating system

Windows, ArchLinux

### Application version

everything above Element Nightly 2023062301

### How did you install the app?

AUR and https://element.io/download for Windows

### Homeserver

archworks.co

### Will you send logs?

Yes

**Comment by robintown** [comment]
Yes, that's what regressed it. We're aware of what we need to do to fix this, it's just outstanding work since the switch to LiveKit. I believe from the VoIP team's perspective we're okay with leaving this broken for a little while, since Element Call is in Labs.
